### PR TITLE
Internet health landing and subpages PR 4798

### PR DESCRIPTION
**Pull request by peiying2**
- Add two new subpages
- Update landing pages
- Update an existing page

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo, there is a navigation page  that contain 5 common strings (the topics).  Should I add them to the "main.lang" file or should I repeat the same strings in the 5 subpages? If they go to the main.lang, I will remove them from the subpages. Then we need to copy them from the localized files to the main.lang file.

**Comment by flodolo** [comment]
My problem with exposing these 5 strings to all locales in main.lang is that we only have a handful of languages translating these pages. As painful as it is, they should stay in each file.

**Comment by flodolo** [comment]
I suggest to cover also these PRs here. Will check the IH pages in the morning
https://github.com/mozilla/bedrock/pull/4783
https://github.com/mozilla/bedrock/pull/4787
https://github.com/mozilla/bedrock/pull/4820

File: en-US/mozorg/internet-health.lang [hunk 1]
```diff
@@ -64,6 +64,7 @@ Help keep the Web in the hands of many
 When just a few organizations and governments control the majority of online content and access, the vital flow of ideas and knowledge is blocked. We champion an internet that belongs to the people.
 
 
+# String is obsolete. Do NOT remove.
```
Comment (flodolo) [inline en-US/mozorg/internet-health.lang]: The string has been removed, it's not obsolete. We mark it as obsolete when it's replaced by a string behind a tag. You should remove both strings from this file.

File: en-US/mozorg/internet-health/digital-inclusion.lang [hunk 2]
```diff
@@ -1,3 +1,4 @@
+## inclusion-donate-tech ##
```
Comment (flodolo) [inline en-US/mozorg/internet-health/digital-inclusion.lang]: There's another tag added to that page

File: en-US/mozorg/internet-health/digital-inclusion.lang [hunk 3]
```diff
@@ -14,12 +15,20 @@ Digital inclusion means promoting diversity, practicing respect, and supporting
...
+Decentralization
+
+
 ;Digital Inclusion
 Digital Inclusion
 
 
-;Open Innovation
-Open Innovation
+;Web Literacy
```
Comment (flodolo) [inline en-US/mozorg/internet-health/digital-inclusion.lang]: This string is behind a tag

File: en-US/mozorg/internet-health/digital-inclusion.lang [hunk 4]
```diff
@@ -14,12 +15,20 @@ Digital inclusion means promoting diversity, practicing respect, and supporting
 Privacy &amp; Security
 
 
+;Open Innovation
+Open Innovation
+
+
+;Decentralization
```
Comment (flodolo) [inline en-US/mozorg/internet-health/digital-inclusion.lang]: This string is behind a tag

File: en-US/firefox/channel/index.lang [hunk 5]
```diff
@@ -60,6 +61,12 @@ Try New Features in a Pre-Release Android Browser | Firefox
...
 
 
+## TAG: firefox_title_meta_bug1357373
+# HTML Description, on the Android page only.
+;Experience cutting-edge features in a pre-release browser for Android: Firefox Beta and Firefox Nightly. Install now!
+Experience cutting-edge features in a pre-release browser for Android: Firefox Beta and Firefox Nightly. Install now!
+
+
 ## TAG: firefox_title_meta_bug1345338
 # HTML Description, on the Android page only.
```
Comment (flodolo) [inline en-US/firefox/channel/index.lang]: This string become obsolete too.

**Comment by flodolo** [comment]
https://github.com/mozilla/bedrock/pull/4787/files also removes two strings, you've only updated the link

File: en-US/download_button.lang [hunk 6]
```diff
@@ -46,6 +50,14 @@ Get it free from the App Store
 Firefox Beta
 
 
+;<span>Firefox</span> Developer Edition
```
Comment (flodolo) [inline en-US/download_button.lang]: In the past we decided to keep this string out of the file on purpose. There's no "for" to localize, and the name is not supposed to be localized. 

I'm OK with exposing "Firefox Nightly", I would leave DevEdition out.

**Pull request merged**